react-frontend/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.7.14",
```

Comment:
should we use apollo client or urql? I like urql's simplicity and have heard some stories around apollo's caching complexity. Thoughts?

Reply:
I don't have a strong preference, happy to try both